server/fidelity/content_api_rss.py

```diff
@@ -14,11 +16,23 @@
 FEATUREMEDIA = "featuremedia"
 WEB_RENDITION = "baseImage"
 CATEGORIES = ("subject_custom",)
+PERMALINK = "Perma_URL"
+BASE_URL = "https://www.fidelityinstitutional.com/"
 
 
 def get_permalink(item):
-    return "{}/{}".format(
-        flask.url_for("rss.index", _external=True).rstrip("/"), item["_id"]
+    code = item["_id"][-6:]
+    try:
+        title = item["extra"][PERMALINK] or ""
+        slug = slugify(title)
+    except (KeyError, AttributeError):
+        title = ""
```

Comment:
Why do we need an empty title variable here?

Reply:
they don't set it for translated items